### PR TITLE
New Vox pressure suits for different jobs

### PR DESCRIPTION
**Pull request by ghost**
Examples:
2015-01-15 14_54_34-orbital trade post xx
voxamplev2

Engineering, Science, and Medical are mostly done. Civ, Supply, Mining, and Sec are still WIP.

Totally mergeable.

### CODE REVIEW

File: code/modules/clothing/spacesuits/alien.dm [hunk 1]
```diff
@@ -141,9 +143,217 @@
...
+
 // Vox Casual
 // Civvie
-/obj/item/clothing/suit/space/vox/casual
+/obj/item/clothing/suit/space/vox/civ
+	name = "vox assistant pressure suit"
+	desc = "A cheap and oddly-shaped pressure suit made for vox crewmembers."
+	icon_state = "vox-civ-assistant"
+	item_state = "vox-pressure-normal"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|FEET|ARMS|HANDS
```
Comment (ComicIronic) [inline code/modules/clothing/spacesuits/alien.dm]: There's really no need to copy all these flags.
They should be inherited from suit/space or suit/space/vox, and they make code harder to edit.

**Comment by Cipherature** [comment]
Alright, cleaned up a few redundancies and added comments as to why some are there.

**Comment by ComicIronic** [comment]
:+1:, some cool designs. Better than ol' porthole, anyways.

My only worry is any confusion between syndi vox and sec vox suits, but that's not really likely.

**Comment by wild-billy** [comment]
:+1:

**Comment by d3athrow** [comment]
Since you are in there may I make a suggestion and make the suits NOT spaceworthy. One of the major complaints about vox is they get a spacesuit off the bat. Light pressure resistance ala firefighter suit would be ok with me though.

**Comment by Cipherature** [comment]
@d3athrow I can do that sure, but I think that leaves the vox without any spaceworthiness whatsoever.

**Comment by ComicIronic** [comment]
That's a good point as well.
We should also consider that balance in terms of plasmen, who get a free suit, with the job bonuses, because they can't take it off. Maybe we should consider giving Vox pressure damage in regular environments to balance the spacesuit.

**Comment by Cipherature** [comment]
For what it's worth, I'd like a poll or something with if it should be changed or not. Making them non-spaceworthy creates quite a few problems that I'm not sure would get solved very quickly.

**Comment by ririchiyo** [comment]
one day we'll have a customizable hardsuit system.

**Comment by ririchiyo** [comment]
Likely conflicted by #2543

**Comment by Cipherature** [comment]
Dang, would have been easier to merge this one first.

**Comment by Cipherature** [comment]
Manually merged the icons back in.

**Comment by Cipherature** [comment]
Should I make more changes to this? Or is there something else that needs to be done?

**Comment by ComicIronic** [comment]
No, it's just up to @D3athrow to decide to merge it.

**Pull request merged**